src/content/en/2020/seo.md

```diff
@@ -83,620 +83,714 @@ Response Code | All Devices | Mobile | Desktop
-
-Analysts for the Web Almanac saw marked different performance distribution scores between the versions.  Refer to the Performance Chapter of the Web Almanac for an in depth comparison between versions 5 and 6.
-
-In tests using v5 of Lighthouse, 15.44% of pages tested scored at or above the 'passing' 85% score. Tests using Version 6 saw only 8.39% of tests achieve a passing score.
-
-<figure markdown>
-  | Percentage of v5 Tests | Percentage of v6 Tests
--- | -- | --
-Good | 15.44% | 8.39%
-Average | 25.49% | 20.19%
-Poor | 59.06% | 71.42%
-
-<figcaption>{{ figure_link(caption="Good, Average and Poor ratios of Lighthouse v5 versus v6", sheets_gid="692150551", sql_file="TODO.sql") }}</figcaption>
-</figure>
+#### Core Web Vitals experiences per device
 
-It is important to note that two of the three new metrics in v6 make up 50% of the weighted performance score. This change in focus sets new, more refined goals. Overall, most pages saw minimal impact with 83.32% of tests shifting ten points or less on the shift to v6.
-
-{{ figure_markup(
-  image="seo-lighthouse-v5-vs-v6.png",
-  caption="Lighthouse Performance score difference between versions 5 and 6",
-  description="Horizontal bar chart showing the score difference between Lighthouse V5 and V6. In tests using v5 of Lighthouse, 15.44% of pages tested scored at or above the 'passing' 85% score. Tests using Version 6 saw only 8.39% of tests achieve a passing score.",
-  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=976167005&format=interactive",
-  sheets_gid="1909295182",
-  sql_file="lighthouse.sql"
-  )
-}}
-
-Lighthouse v6 and with it the integration of Core Web Vitals rolled out across Google products with the release of Chrome 84 in July 2020. Chrome DevTools Audits panel was renamed to Lighthouse. Pagespeed insights and Google Search Console also reference these unified metrics.
-
-More information on the Lighthouse performance testing results and score details is available in the Performance section.
-
-#### Web vitals and other CrUX Metrics per Device
-
-Desktop continues to be the more performant platform for users despite more users on mobile devices. 33.13% of websites scored 'Good' Core Web Vitals for desktop while only 19.96% of their mobile counterparts passed the Core Web Vitals assessment.
+Desktop continues to be the more performant platform for users despite more users on mobile devices. 33.13% of websites scored _Good_ Core Web Vitals for desktop while only 19.96% of their mobile counterparts passed the Core Web Vitals assessment.
 
+{# TODO(analysts): Please double check the following two sql_files, as these metrics are related to Lighthouse. #}
```

Comment:
@antoineeripret @Tiggerito

Reply:
Good spot @rviscomi . That chart did not come from lighthouse.sql. It's in:

https://docs.google.com/spreadsheets/d/1ram47FshAjzvbQVJbAQPgxZN7PPOPCKIK67VJZCo92c/edit#gid=996380787

I think @fellowhuman1101 did that one?

I now have 20 Github tabs, 10 drive tabs, and 3 sheets tabs open. And 15 tabs in my Visual Studio Code. Hopefully no more needed!

Reply:
@rviscomi @Tiggerito The data is from the Performance results workbook (Tab: Web Vitals per device) >> https://docs.google.com/spreadsheets/d/164FVuCQ7gPhTWUXJl1av5_hBxjncNi0TK8RnNseNPJQ/edit#gid=1270303192&range=A1

Copied into the SEO chapter to create the charts.

Reply:
@rviscomi can I reference this by: ../09_Performance/web_vitals_by_device.sql

Reply:
I've updated the sql reference for this chart and the one after using a relative path